### PR TITLE
 chore(github): add some tips to improve issues and PRs

### PR DESCRIPTION
**Pull request by kevinawoo**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -38,3 +38,39 @@ Descriptions:
...
+
+## Put a border around a screenshot/gif
+Add a border to help improve the readability.
+```html
+<kbd>![](https://www.spinnaker.io/assets/images/spinnaker-logo-inline.svg)</kbd>
+```
+
+## Add expandable sections
+<details>
+<summary>Add expandable sections to hide away large code blocks/images or reduce the motion of a gifs</summary>
```
Comment (anotherchrisberry) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
<summary>Add expandable sections to hide away large code blocks/images or reduce the motion of a gif</summary>
```

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -38,3 +38,39 @@ Descriptions:
...
+---
+# Helpful Tips
+
+## Add screenshots and annotations
+<kbd>![](https://cl.ly/744faf8f96d7/download/Image%202019-04-08%20at%2012.35.32.png)</kbd>
+- In OSX Mojave, you can capture screenshots and add annotations. https://support.apple.com/en-us/HT201361
+- Another app you can use is https://www.getcloudapp.com/.
+
+## Add a gif to show an action
+GIFs are great to show an action, however they sometimes move too fast.
```
Comment (anotherchrisberry) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
Animated gifs are a great way to show action; however, they sometimes move too fast.
```
(getting around inconsistent capitalization of `gif`)

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -38,3 +38,39 @@ Descriptions:
...
+<kbd>![](https://cl.ly/744faf8f96d7/download/Image%202019-04-08%20at%2012.35.32.png)</kbd>
+- In OSX Mojave, you can capture screenshots and add annotations. https://support.apple.com/en-us/HT201361
+- Another app you can use is https://www.getcloudapp.com/.
+
+## Add a gif to show an action
+GIFs are great to show an action, however they sometimes move too fast. 
+One way to improve them is to capture 3 pieces of content:
+- screenshot before an action
+- screenshot after the action
+- gif of the action
```
Comment (anotherchrisberry) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
- animated gif of the action
```

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -38,3 +38,39 @@ Descriptions:
...
+
+---
+# Helpful Tips
+
+## Add screenshots and annotations
+<kbd>![](https://cl.ly/744faf8f96d7/download/Image%202019-04-08%20at%2012.35.32.png)</kbd>
+- In OSX Mojave, you can capture screenshots and add annotations. https://support.apple.com/en-us/HT201361
+- Another app you can use is https://www.getcloudapp.com/.
+
+## Add a gif to show an action
```
Comment (anotherchrisberry) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
## Add an animated gif to show an action
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 5]
```diff
@@ -18,3 +18,38 @@
...
+---
+# Helpful Tips
+
+## Add screenshots and annotations
+<kbd>![](https://cl.ly/744faf8f96d7/download/Image%202019-04-08%20at%2012.35.32.png)</kbd>
+- In OSX Mojave, you can capture screenshots and add annotations. https://support.apple.com/en-us/HT201361
+- Another app you can use is https://www.getcloudapp.com/.
+
+## Add a gif to show an action
+GIFs are great to show an action, however they sometimes move too fast.
```
Comment (anotherchrisberry) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
Animated gifs are a great way to show action; however, they sometimes move too fast.
```

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 6]
```diff
@@ -18,3 +18,38 @@
...
+
+---
+# Helpful Tips
+
+## Add screenshots and annotations
+<kbd>![](https://cl.ly/744faf8f96d7/download/Image%202019-04-08%20at%2012.35.32.png)</kbd>
+- In OSX Mojave, you can capture screenshots and add annotations. https://support.apple.com/en-us/HT201361
+- Another app you can use is https://www.getcloudapp.com/.
+
+## Add a gif to show an action
```
Comment (anotherchrisberry) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
## Add an animated gif to show an action
```

File: .github/ISSUE_TEMPLATE.md [hunk 7]
```diff
@@ -38,3 +38,39 @@ Descriptions:
...
+GIFs are great to show an action, however they sometimes move too fast. 
+One way to improve them is to capture 3 pieces of content:
+- screenshot before an action
+- screenshot after the action
+- gif of the action
+- You can edit a gif using https://ezgif.com/
```
Comment (anotherchrisberry) [inline .github/ISSUE_TEMPLATE.md]: Should this be "create" instead of "edit"?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 8]
```diff
@@ -18,3 +18,38 @@
...
+GIFs are great to show an action, however they sometimes move too fast. 
+One way to improve them is to capture 3 pieces of content:
+- screenshot before an action
+- screenshot after the action
+- gif of the action
+- You can edit a gif using https://ezgif.com/
```
Comment (anotherchrisberry) [inline .github/PULL_REQUEST_TEMPLATE.md]: "create"?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 9]
```diff
@@ -18,3 +18,38 @@
...
+<kbd>![](https://cl.ly/744faf8f96d7/download/Image%202019-04-08%20at%2012.35.32.png)</kbd>
+- In OSX Mojave, you can capture screenshots and add annotations. https://support.apple.com/en-us/HT201361
+- Another app you can use is https://www.getcloudapp.com/.
+
+## Add a gif to show an action
+GIFs are great to show an action, however they sometimes move too fast. 
+One way to improve them is to capture 3 pieces of content:
+- screenshot before an action
+- screenshot after the action
+- gif of the action
```
Comment (anotherchrisberry) [inline .github/PULL_REQUEST_TEMPLATE.md]:
```suggestion
- animated gif of the action
```

File: .github/ISSUE_TEMPLATE.md (hunk 7 above)
Comment (kevinawoo) [reply .github/ISSUE_TEMPLATE.md]: create/edit?

I've been recording a gif on my computer, then editing it on ezgif.com

Comment (cfieber) [reply .github/ISSUE_TEMPLATE.md]: in this context I'd agree with @anotherchrisberry that create is right

**Comment by ezimanyi** [comment]
I'm wondering if it might make more sense to add this info to our spinnaker.io site and link to it in the pull request template.  I think this is super helpful info, but I'm not sure that having it pre-populated into each PR/issue is necessarily the best place for it.  Partly this is because it's reasonably common for people to forget to delete the template, and I'd like to avoid making the description even long in those cases.

**Pull request closed without merging**